uint32 recordingTime
---
string recordedFileDest
